### PR TITLE
revert: patch 9.1.0503

### PR DESCRIPTION
**Pull request by glepnir**
fuzzy can handle keyword completion as well. so fuzzycollect is no longer needed.

### CODE REVIEW

File: runtime/doc/version9.txt [hunk 1]
```diff
@@ -41642,9 +41642,6 @@ Improved visual highlighting.
 
 Python3 support in OpenVMS.
 
-Support for |fuzzy-matching| during |ins-completion| with the "fuzzy" and
-"fuzzycollect" values of the 'completeopt' setting
-
```
Comment (zeertzjq) [inline runtime/doc/version9.txt]: You should just remove "fuzzycollect" from the sentence instead of deleting the whole sentence.

Comment (glepnir) [reply runtime/doc/version9.txt]: oops sry.

File: runtime/doc/version9.txt [hunk 2]
```diff
@@ -41642,8 +41642,7 @@ Improved visual highlighting.
 
 Python3 support in OpenVMS.
 
-Support for |fuzzy-matching| during |ins-completion| with the "fuzzy" and
-"fuzzycollect" values of the 'completeopt' setting
+Support for |fuzzy-matching| during |ins-completion| with the "fuzzy".
```
Comment (zeertzjq) [inline runtime/doc/version9.txt:41645]:
```suggestion
Support for |fuzzy-matching| during |ins-completion| with the "fuzzy"
value of the 'completeopt' option.
```

**Comment by chrisbra** [comment]
thanks!

**Comment by zeertzjq** [comment]
This change from patch 9.1.0503 hasn't been reverted:
```diff
diff --git a/src/insexpand.c b/src/insexpand.c
index c673df927..78fea5127 100644
--- a/src/insexpand.c
+++ b/src/insexpand.c
@@ -4100,8 +4100,7 @@ find_comp_when_fuzzy(void)
     int		is_backward = compl_shows_dir_backward();
     compl_T	*comp = NULL;
 
-    if (compl_match_array == NULL ||
-	    (is_forward && compl_selected_item == compl_match_arraysize - 1)
+    if ((is_forward && compl_selected_item == compl_match_arraysize - 1)
 	    || (is_backward && compl_selected_item == 0))
 	return compl_first_match;
 
```
Does it need to be?

**Comment by glepnir** [comment]
find_next_completion_match has checked the array is not NULL so there no need revert.

**Pull request closed without merging**